ViAn/GUI/mainwindow.cpp

```diff
@@ -232,25 +237,55 @@ void MainWindow::resizeEvent(QResizeEvent* event) {
  * @param newPos current position of the slider
```

Comment:
Maybe update this comment so that it describes more about what is dane in the function.